src/types/config.js

```diff
@@ -11,6 +11,7 @@
  * @prop {string} apiUrl
  * @prop {string} authority
  * @prop {string} grantToken
+ * @prop {string} icon
```

Comment:
The `icon` property is optional. See https://github.com/hypothesis/client/blob/c8df07a364061d82d24c61da9fcd9ba6d1b67b1f/src/sidebar/components/group-list.js#L23.